scss/_orange-footer.scss

```diff
@@ -0,0 +1,209 @@
+.footer-mailing {
+  padding-top: $footer-orange-mailing-padding-top;
+  padding-bottom: $footer-orange-mailing-padding-bottom;
+
+  @include media-breakpoint-up(md) {
+    padding-bottom: $footer-orange-mailing-padding-bottom-md;
+  }
+
+  h3 {
```

Comment:
Not sure it's a good way to apply style to h3, maybe there's a need to create a `.footer-heading` class? This class could be apply on h2, h3, h4 depending on website heading hierarchy?
Avoid fixing the font-size and maybe add into documentation sample without heading, and another with h3 heading + some classes to reset to normal font-size?